assets/app/views/browse/build-config.html

```diff
@@ -22,7 +22,7 @@
                 <button
                     class="btn btn-default hidden-xs"
                     ng-click="startBuild()"
-                    ng-disabled="!canBuild">
+                    ng-disabled="!canBuild()">
```

Comment:
meh im not a fan of having function calls in our templates for attribute checks. is there a reason we couldn't properly update the value when the different bits of data load?